src/ui/layouts/common/layouts/AuthenticatedLayout/DeploymentBanner/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './styles.module.scss';
+
+function DeploymentBanner() {
+  return (
+    <aside className={styles.deploymentBanner}>
+      <p className={styles.deploymentBanner__paragraph}>
+        You are running ZenML locally. In order to run MLOps workloads in the
+        cloud, you first need to <pre>deploy ZenML</pre>. Check the{' '}
+        <a
+          target="_blank"
+          rel="noopener noreferrer"
+          href="https://docs.zenml.io/getting-started/deploying-zenml"
+        >
+          {' '}
+          docs
+        </a>{' '}
+        to see how.
+      </p>
+    </aside>
```

Comment:
Alternate Formulation that I would suggest here. 
```suggestion
    <aside className={styles.deploymentBanner}>
      <p className={styles.deploymentBanner__paragraph}>
        ZenML is running locally. In order to run your pipelines in the
        cloud, you need to <pre>deploy ZenML</pre>. See the{' '}
        <a
          target="_blank"
          rel="noopener noreferrer"
          href="https://docs.zenml.io/getting-started/deploying-zenml"
        >
          {' '}
          docs
        </a>{' '}
        to learn how.
      </p>
    </aside>
```